### ISSUE TITLE
Accessibility: Image link is missing alternative text

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#422]
**Description**
It looks as though this issue appears on all images on the site that are on Feature pages. The links all seem to be those around the Items, taking users to that Item within the exhibit. [Example report for Irene Castel Corticelli Fashions](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=23020461180&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/2.4.4/Check/5).

**Solutions**
Option 1: If all images have alt text that describe the location that the link directs the user to, it would meet compliance. Image links should always say where the link goes, rather than describing the image itself.
Option 2: Add text with each image that would say where the link goes. This could be set with a screen-reader-only class so that it doesn't show for sighted users but would be read for non-sighted users.

I would recommend referring to [this page (WCAG 2.0 H30)](https://www.w3.org/TR/WCAG20-TECHS/H30.html) for how to fix this issue.
